Serverless FAQ and limitations (#11376)
pingcap/docs: tidb-cloud/serverless-tier-faqs.md

---
title: Serverless Tier FAQs
summary: Learn about the most frequently asked questions (FAQs) relating to TiDB Cloud Serverless Tier.
---

# Serverless Tier FAQs

<!-- markdownlint-disable MD026 -->

This document lists the most frequently asked questions about TiDB Cloud Serverless Tier.

## General FAQs

### What is Serverless Tier?

TiDB Cloud Serverless Tier delivers the free starter TiDB clusters with full HTAP capabilities for you and your organization. It is a fully managed, auto-scaling deployment of TiDB that lets you start using your database immediately, develop and run your application without caring about the underlying nodes, and automatically scale based on your application's workload changes.

### How do I get started with Serverless Tier?

Get started with the 5-minute [TiDB Cloud Quick Start](/tidb-cloud/tidb-cloud-quickstart.md).

> ### What are the limitations of a Serverless Tier cluster?

Reviewer: 这里建议 link 到 tidb-cloud/serverless-tier-limitations.md

Author: 下面有link呀。

- For each TiDB Cloud account, you can create one complimentary Serverless Tier cluster to use during the beta phase. You need to delete the existing Serverless Tier cluster then create a new one.
- Each Serverless Tier cluster has the following limitations:
    - The storage size is limited to 5 GiB (logical size) of OLTP storage and 5 GiB of OLAP storage.
    - The compute resource is limited to 1 vCPU and 1 GiB RAM.
- There are some features that Serverless Tier does not support or partially supports. See [Serverless Tier Limitations](/tidb-cloud/serverless-tier-limitations.md) for details.

### Is Serverless Tier free during beta?

> Yes. Serverless Tier cluster is free to use during beta phase. When Serverless Tier goes GA, we will still offer a free starter plan and have a usage-based billing plan for additional resources and maintain higher performance.

Author: 我又改了一下

Get started with the 5-minute [TiDB Cloud Quick Start](/tidb-cloud/tidb-cloud-quickstart.md).

### Is Serverless Tier free during beta?

Yes. Serverless Tier is free to use during the beta phase. In the coming months, we intend to offer a usage-based billing plan for additional resources and higher performance, while still keeping offering the free starter plan.

### What does it mean for beta release?

Serverless Tier is in beta while we continuously add new features and improve existing features before it becomes generally available. We do not provide SLA for beta products. Therefore, Serverless Tier should **NOT** be used in production currently.

### What are the limitations of a Serverless Tier cluster in free beta?

- For each TiDB Cloud account, you can create one complimentary Serverless Tier cluster during the beta phase. To create a new Serverless Tier cluster, you need to delete the existing one first.
- Each Serverless Tier cluster has the following limitations:
    - The storage size is limited to 5 GiB (logical size) of OLTP storage and 5 GiB of OLAP storage.
    - The compute resource is limited to 1 vCPU and 1 GiB RAM.

>     - **Note**: In the coming months, we intend to offer a usage-based billing plan for additional resources and higher performance, while still keeping offering the free starter plan. In the coming releases, the limitations of the free serverless tier might be changed.

Reviewer: ```suggestion
    - **Note**: In the coming months, we intend to offer a usage-based billing plan for additional resources and higher performance, while still keeping offering the free starter plan. In the coming releases, the limitations of the free Serverless Tier might be changed.
```

>     - **Note**: In the coming months, we intend to offer a usage-based billing plan for additional resources and higher performance, while still keeping offering the free starter plan. After the coming releases, the limitation of the free serverless tier may be changed.

Author: ```suggestion
    - **Note**: In the coming months, we intend to offer a usage-based billing plan for additional resources and higher performance, while still keeping offering the free starter plan. In the coming releases, the limitations of the free serverless tier might be changed.
```

### I created a Developer Tier cluster before Serverless Tier was available. Can I still use my cluster?

Yes, your free Developer Tier cluster will be automatically migrated to the Serverless Tier cluster soon. Your ability to use your cluster should not be affected, and you will have the same improved Serverless Tier user experiences.

## Security FAQs

### Is my Serverless Tier shared or dedicated?

The serverless technology is designed for multi-tenancy and the resources used by all clusters are shared. To get managed TiDB service with isolated infrastructure and resources, you can upgrade it to the [Dedicated Tier](/tidb-cloud/select-cluster-tier.md#dedicated-tier).

### How does TiDB Serverless Tier ensure security?

- Your connections are encrypted by Transport Layer Security (TLS). For more information about using TLS to connect to Serverless Tier, see [Secure Connections to Serverless Tier Clusters](/tidb-cloud/secure-connections-to-serverless-tier-clusters.md).

> - All persisted data on Serverless Tier is encrypted-at-rest using the cloud provider's tool that your cluster is running in.

Reviewer: ```suggestion
- All persisted data on Serverless Tier is encrypted-at-rest using the tool of the cloud provider that your cluster is running in.
```

## Maintenance FAQ

### Can I upgrade the version of TiDB that my cluster is running on?

> No, Serverless Tier clusters are upgraded automatically as we release new TiDB versions on TiDB Cloud. You can see what version of TiDB your cluster is running in the [TiDB Cloud Console](https://tidbcloud.com/console/clusters) or in the latest [release note](https://docs.pingcap.com/tidbcloud/release-notes). Alternatively, you can also connect to your cluster and use `SELECT version()` or `SELECT tidb_version()` to check the TiDB version.

Reviewer: ```suggestion
No. Serverless Tier clusters are upgraded automatically as we roll out new TiDB versions on TiDB Cloud. You can see what version of TiDB your cluster is running in the [TiDB Cloud Console](https://tidbcloud.com/console/clusters) or in the latest [release note](https://docs.pingcap.com/tidbcloud/release-notes). Alternatively, you can also connect to your cluster and use `SELECT version()` or `SELECT tidb_version()` to check the TiDB version.
```
